dist/scripts/NotificationSystem/actions/index.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
Looks like stuff isn't getting bundled anymore. All these scripts should be under `rhinostyle.min.js` and `rhinostyle.min.js.map`. There should be no source code in the `dist` folder.